config/config.yaml

```diff
@@ -11,3 +11,5 @@ cerberus:
 
 tunings:
     wait_duration: 60                                      # Duration to wait between each chaos scenario
+    iterations: 2                                          # Number of times to execute the scenarios
```

Comment:
Maybe set the number of iterations to 1?

Reply:
+1 to 1 iterations as a default

Reply:
@chaitanyaenr  @mffiedler Is it ok to have the default for the daemon mode variable to True? or should I set it to false to only run 1 iteration by default?

Reply:
I think daemon=false and 1 iteration is the least amount of surprise for someone running kraken for the first time.   Almost "demo" mode.

Reply:
+1 for setting daemon_mode to false.